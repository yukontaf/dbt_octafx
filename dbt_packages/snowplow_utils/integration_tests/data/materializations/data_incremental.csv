run,id,start_tstamp
1,1,2021-01-01 00:00:00
1,2,2021-03-01 00:00:00
1,2,2021-03-03 00:00:00
1,3,2021-03-03 00:00:00
1,4,2021-03-04 00:00:00
2,1,2021-03-05 00:00:00
2,2,2021-03-02 00:00:00
2,3,2021-03-07 00:00:00
2,5,2021-03-08 00:00:00
