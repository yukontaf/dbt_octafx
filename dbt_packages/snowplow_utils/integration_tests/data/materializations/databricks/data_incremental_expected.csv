id,start_tstamp
1,2021-01-01 00:00:00
1,2021-03-05 00:00:00
2,2021-03-02 00:00:00
2,2021-03-02 00:00:00
3,2021-03-07 00:00:00
4,2021-03-04 00:00:00
5,2021-03-08 00:00:00
